pkg/logger/logger.go

```diff
@@ -0,0 +1,167 @@
+package logging
```

Comment:
I feel like I'm missing something; this is unused, right? If I'm not mistaken, it probably belongs in a different PR.

I did not review this file in depth; LMK if I've got it wrong and I'll take a closer look.

Reply:
Yeah I was going to use it in my PR but never got around to it so should be deleted. It belongs in a bigger refactor anyway